docker/Dockerfile.tests

```diff
@@ -0,0 +1,29 @@
+# PYTHON_VERSION in ["3.7", "3.8", "3.9"]
+ARG PYTHON_VERSION=3.8
+# SOURCE in ["local", "github"]
+ARG SOURCE=local
+
+# Our base image is the official python image. If needed we can look into slimming this down.
+FROM python:${PYTHON_VERSION} AS base
+RUN apt update && apt install default-jre unixodbc unixodbc-dev locales locales-all -y && apt clean
+RUN python -m pip install --upgrade pip
+
+FROM base as build_local
+WORKDIR /great_expectations
+COPY . .
+
+FROM base AS build_github
+# BRANCH is the branch name you want to build
+ARG BRANCH=develop
+RUN git clone --depth 1 --branch ${BRANCH} https://github.com/great-expectations/great_expectations.git
+WORKDIR /great_expectations
+
+FROM build_${SOURCE} AS dev
+RUN pip install --requirement requirements.txt --requirement requirements-dev.txt \
```

Comment:
You should be able to just install the requirements-dev.txt file since that references all the other files that are explicitly listed

Reply:
Thanks for pointing this out! I was following what we currently do in the `azure-pipelines.yml` file but hadn't looked to closely at the contents of these files. I'll give them a look and update. Thanks!

Reply:
Done.